config/sync/automated_cron.settings.yml

```diff
@@ -1 +1 @@
-interval: 10800
+interval: 3600
```

Comment:
I don’t think this is a good idea. Performance hits. Ultimate corn is handling most of the corn jobs anyway.